app/javascript/textarea-markdown-linkify.js

```diff
@@ -0,0 +1,37 @@
+import MarkdownIt from 'markdown-it'
+
+export default class {
+  linkify(selector) {
+    const textareas = document.querySelectorAll(selector)
+    if (textareas.length === 0) {
+      return null
+    }
```

Comment:
```suggestion
    if (!textareas.length) return
```

好みがわかれそうだけど、こうするとか？（lintに怒られるかな？）

少なくとも、nullって明示的に返す必要があるんでしょうか？returnだけでよくないですか？（これもlintの指摘？）

Reply:
lintも問題なく、returnだけでも問題なさそうなので、提示いただいたコードに修正しました。[b83490b](https://github.com/fjordllc/bootcamp/pull/6747/commits/b83490b3e2fae954e8ded00415d4a4ef555ca8dc)